web/ui/react-app/src/pages/graph/GraphHelpers.ts

```diff
@@ -195,3 +230,27 @@ export const parseValue = (value: string) => {
   // can't be graphed, so show them as gaps (null).
   return isNaN(val) ? null : val;
 };
+
+const exemplarSymbol = (ctx: CanvasRenderingContext2D, x: number, y: number) => {
+  // Center the symbol on the point.
+  y = y - 3.5;
+
+  // Correct if the symbol is overflowing off the grid.
+  if (x > ctx.canvas.clientWidth - 59) {
+    x = ctx.canvas.clientWidth - 59;
+  }
+  if (y > ctx.canvas.clientHeight - 40) {
+    y = ctx.canvas.clientHeight - 40;
```

Comment:
I had to change these values to keep the symbols from overflowing off the bottom, but other than that it's a direct copy of your comment. Looks really good!